packages/terra-collapsible-menu-view/tests/wdio/collapsible-menu-view-spec.js

```diff
@@ -31,6 +31,30 @@ Terra.describeViewports('Responsive Single Item', ['tiny', 'large'], () => {
     browser.moveToObject('#root', 0, 0);
   });
 
+  Terra.viewports('enormous', 'huge').forEach((viewport) => {
```

Comment:
Re-run this to update the screenshots after the changes in "SingleItemGroup.test.jsx"